apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.ts

```diff
@@ -161,13 +165,7 @@ router.post(
           ),
         );
       } else {
-        if (req.get('HX-Request')) {
-          res
-            .set('HX-Redirect', res.locals.urlPrefix + '/jobSequence/' + result.jobSequenceId)
-            .send();
-        } else {
-          res.redirect(res.locals.urlPrefix + '/jobSequence/' + result.jobSequenceId);
-        }
+        res.send(GenerationFailure(res.locals.urlPrefix, result.jobSequenceId));
```

Comment:
This only updates the regeneration flow; how about for initial generations?

Reply:
Done in 4c35422.